Generals/Code/Libraries/Source/WWVegas/WW3D2/dx8wrapper.h

```diff
@@ -826,14 +830,8 @@ WWINLINE unsigned int DX8Wrapper::Convert_Color(const Vector3& color,float alpha
 
 WWINLINE void DX8Wrapper::Clamp_Color(Vector4& color)
```

Comment:
In ZH the spelling mistake above with "vertor" was fixed. You can do this here too.